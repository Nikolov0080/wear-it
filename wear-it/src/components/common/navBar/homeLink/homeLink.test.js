import React from 'react';
import renderer from 'react-test-renderer';
import HomeLink from './homeLink';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Home Link', () => {
    const tree = renderer
        .create(<BrowserRouter><HomeLink /></BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});