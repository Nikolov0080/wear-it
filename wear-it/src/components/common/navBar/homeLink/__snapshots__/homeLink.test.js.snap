// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Home Link 1`] = `
<div
  className="homeBtn"
>
  <div
    className="box"
  >
    <a
      className="linkBtn"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
  </div>
</div>
`;
